.github/workflows/ci.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       uses: shivammathur/setup-php@2.9.0
       with:
         php-version: ${{ matrix.php }}
+        tools: phpunit
+    - run: echo 'xdebug.max_nesting_level=3000' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
```

Comment:
I actually meant like [this](https://github.com/shivammathur/setup-php#ini-values-optional).